Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -896,102 +897,93 @@ void CSequenceBatchRenderDialog::InitializeContext()
-    QString folder = renderItem.folder;
-    QString itemText = m_ui->m_renderList->model()->index(m_renderContext.currentItemIndex, 0).data().toString();
-    itemText.replace('/', '-'); // A full sequence name can have slash characters which aren't suitable for a file name.
-    folder += "/";
-    folder += itemText;
+    AZStd::string folderName;
+    AzFramework::StringFunc::Path::Join(renderItem.folder.toUtf8().data(), renderListNameSanitized.toUtf8().data(), folderName);
 
     // If this is a relative path, prepend the @products@ folder to match where the Renderer is going
     // to dump the frame buffer image captures.
-    if (AzFramework::StringFunc::Path::IsRelative(folder.toUtf8().data()))
+    if (AzFramework::StringFunc::Path::IsRelative(folderName.c_str()))
     {
         AZStd::string absolutePath;
-        AZStd::string assetsRoot = AZ::IO::FileIOBase::GetInstance()->GetAlias("@products@");
-        AzFramework::StringFunc::Path::Join(assetsRoot.c_str(), folder.toUtf8().data(), absolutePath);
-        folder = absolutePath.c_str();
+        const AZStd::string assetsRoot = AZ::IO::FileIOBase::GetInstance()->GetAlias("@products@");
+        AzFramework::StringFunc::Path::Join(assetsRoot.c_str(), folderName.c_str(), absolutePath);
+        folderName = absolutePath;
     }
 
-    QString finalFolder = folder;
+    QString finalFolder = folderName.c_str();
     int i = 2;
-    while (QFileInfo::exists(finalFolder))
+    QString probeName = finalFolder;
+    while (QFileInfo::exists(probeName))
     {
-        finalFolder = folder;
-        const QString suffix = QString::fromLatin1("_v%1").arg(i);
-        finalFolder += suffix;
-        ++i;
+        probeName = QObject::tr("%1_v%2").arg(finalFolder).arg(i++);
     }
+    finalFolder = probeName;
 
-    // create a new folder before writing any files
-    QDir().mkdir(finalFolder);
-
-    m_renderContext.captureOptions.folder = finalFolder.toUtf8().data();
-
-    // Change the resolution.
-    const int renderWidth = getResWidth(renderItem.resW);
-    const int renderHeight = getResHeight(renderItem.resH);
-    ICVar* pCVarCustomResWidth = gEnv->pConsole->GetCVar("r_CustomResWidth");
-    ICVar* pCVarCustomResHeight = gEnv->pConsole->GetCVar("r_CustomResHeight");
-    if (pCVarCustomResWidth && pCVarCustomResHeight)
+    // Create a new folder before writing any files
+    const bool mkdirResult = QDir().mkdir(finalFolder);
+    if (!mkdirResult)
     {
-        // If available, use the custom resolution cvars.
-        m_renderContext.cvarCustomResWidthBU = pCVarCustomResWidth->GetIVal();
-        m_renderContext.cvarCustomResHeightBU = pCVarCustomResHeight->GetIVal();
-        pCVarCustomResWidth->Set(renderWidth);
-        pCVarCustomResHeight->Set(renderHeight);
+        QMessageBox::critical(AzToolsFramework::GetActiveWindow(), QString(), QObject::tr("Cannot create directory %1 for output frames").arg(finalFolder));
```

Comment:
I think we want to use MessagePopupRequests or something like that to centralize all dialogs. In particular with the batch renderer as might be run on a separate server to do render, and when this happens you want to be able to disable all popups or blocking UI